src/routes/uniswap_v3.ts

```diff
@@ -121,7 +118,7 @@ router.post('/result', async (req: Request, res: Response) => {
   }
 });
 
-router.get('/start', async (req, res) => {
+router.get('/order-pairs', async (req, res) => {
```

Comment:
Changing the route would require changes to the client. The goal of this refactor should be to remove the pairs caching code while minimizing changes to the client needed.